models/topics.model.js

```diff
@@ -0,0 +1,8 @@
+const db = require('../db/connection')
+
+exports.fetchTopics = () => {
+    return db.query(`SELECT * FROM topics`).then(({ rows: topics }) => {
```

Comment:
Love the destructuring with a custom key here, it makes the code super clean and easy to read 👍